lib/status/status.js

```diff
@@ -185,7 +221,10 @@ class Status {
         steps.push(step)
       }
       step.status = status
-      step.output = output
+
+      if (output) {
+        step.output = output
+      }
```

Comment:
this avoids cleaning a step output when a step runs an update without any output (or when the _runStep method does)